### ISSUE TITLE
Moscow

### ISSUE THREAD

**Issue by cmandujanoh** [KaukinenConfed/UmbrellaAcademy#9]
It's a city with great history, there are great architecture materpieces, and I like its weather.

**Comment by jaracenaConfederation** [comment]
Yes, It is a beautiful place to visit. In the pictures, the city looks clean and familiar.

**Comment by sangnln** [comment]
Agree. Moscow is a beautiful city to visit.

**Comment by mihir-934** [comment]
We share a common choice,
